charts/growi/README.md

```diff
@@ -0,0 +1,41 @@
+# GROWI Helm Chart
+
+## TL;DR;
+```bash
+$ helm repo add weseek https://weseek.github.io/helm-charts/
+$ helm install growi-example weseek/growi
+```
+
+## Prerequisites
+* Helm 3.0+
```

Comment:
Elasticsearch helm chart doesn't support Helm 3, so is this OK?

https://github.com/elastic/helm-charts/blob/master/README.md#helm-versions

Reply:
At the moment, https://github.com/elastic/helm-charts/issues/412 is blocking official support.
But it looks like it is not an issue of neither Helm 3 or Elastic's Charts.
https://github.com/kubernetes/kubectl/issues/798